### PR TITLE
New command ipsec order

### PR DESCRIPTION
**Pull request by caberos**
[#issue1696](https://github.com/softlayer/softlayer-python/issues/1696)

image

### CODE REVIEW

File: SoftLayer/CLI/vpn/ipsec/order.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+@click.command(cls=SoftLayer.CLI.command.SLCommand, )
+@click.option('--datacenter', '-d', required=True, prompt=True, help="Datacenter shortname")
+@environment.pass_env
+def cli(env, datacenter):
+    """Order/create a IPSec VPN tunnel instance."""
+
+    ipsec_manager = SoftLayer.IPSECManager(env.client)
+
+    result = ipsec_manager.order(datacenter, ['IPSEC_STANDARD'])
```
Comment (BrianSantivanez) [inline SoftLayer/CLI/vpn/ipsec/order.py:20]: Like ipsec has a cost, I think it is a good idea to add a confirmation before to order
```
        if not (env.skip_confirmations or formatting.confirm(
                "This action will incur charges on your account. Continue?")):
            raise exceptions.CLIAbort('Aborting ipsec order.')
```

**Comment by edsonarios** [comment]
Screenshots of the test result
image
image

File: SoftLayer/managers/ipsec.py [hunk 2]
```diff
@@ -284,3 +285,17 @@ def update_tunnel_context(self, context_id, friendly_name=None,
         if phase2_key_ttl is not None:
             context['phaseTwoKeylife'] = phase2_key_ttl
         return self.context.editObject(context, id=context_id)
+
+    def order(self, datacenter, item_package):
```
Comment (allmightyspiff) [inline SoftLayer/managers/ipsec.py:289]: Add a unit test for this new function

**Review by allmightyspiff — CHANGES_REQUESTED** [review]
Looks good, just need a unit test for the new manager function you added, and a confirmation for the order like Brian mentioned. Thanks

**Review by allmightyspiff — APPROVED** [review]

**Pull request merged**